bolt/src/Passes/InstrumentationSummary.h

```diff
@@ -104,9 +104,9 @@ struct InstrumentationSummary {
   /// Stores function names, to be emitted to the runtime
   std::string StringTable;
 
-  /// Our runtime indirect call instrumenter function
-  MCSymbol *IndCallHandlerFunc;
-  MCSymbol *IndTailCallHandlerFunc;
+  /// Pointer to runtime instrumentation handler
+  MCSymbol *IndCallInstrPointer;
```

Comment:
The "Instr" here confused me a bit because I was associating with "Instr" being "Instruction". Perhaps we can rename to IndCallCounterFuncPtr? (since it is basically just updating a counter)